public/index.html

```diff
@@ -28,8 +44,9 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+    <div class="background-container">
```

Comment:
is this a bootstrap class? If not, what purpose does it serve? I don't see any styling anywhere for it.

Reply:
Used it but deleted the css